arboretum/kubernetes/fetchers/fetch_cluster_resource.py

```diff
@@ -0,0 +1,94 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Cluster resource fetcher for various types of clusters."""
+
+import json
+from collections import defaultdict
+
+from arboretum.common.constants import KUBE_RESOURCE_TYPE_DEFAULT
+
+from compliance.evidence import (
+    DAY, RawEvidence, get_evidence_dependency, store_raw_evidence
+)
+from compliance.fetch import ComplianceFetcher
+
+
+class ClusterResourceFetcher(ComplianceFetcher):
+    """Fetch resources of vanilla kubernetes clusters."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Initialize the fetcher object with configuration settings."""
+        cls.config.add_evidences(
+            [
+                RawEvidence(
+                    'cluster_resource.json',
+                    'kubernetes',
+                    DAY,
+                    'Kubernetes cluster resources'
+                )
+            ]
+        )
+        return cls
+
+    @store_raw_evidence('kubernetes/cluster_resource.json')
+    def fetch_cluster_resource(self):
+        """Fetch cluster resources of listed clusters."""
+        ev = get_evidence_dependency(
+            'raw/auditree/compliance_config.json', self.locker
+        )
+        cluster_list_config = json.loads(ev.content)['org']['kubernetes']
+        cluster_list = cluster_list_config['cluster_list']
```

Comment:
why is this coming from evidence rather than directly from the config?

Reply:
Let's use cluster list in the config.